src/components/personkort/PersonkortChangeEnhet.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from "react";
+import { BodyLong, Button, Heading, Modal } from "@navikt/ds-react";
+import { BehandlendeEnhet } from "@/data/behandlendeenhet/types/BehandlendeEnhet";
+import styled from "styled-components";
+import { useChangeEnhet } from "@/components/personkort/useChangeEnhet";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import ModalWrapper from "nav-frontend-modal";
+
+const texts = {
+  endre: "Endre til",
+  innholdModal:
+    "Hvis du ikke har tilgang til NAV utland, vil du miste tilgangen til denne personen når enheten endres.\n" +
```

Comment:
\n i tekst kan vel ende opp med å se rart ut hvis man endrer størrelsen på skjermen? Det er kanskje bedre å enten dele det i to elementer, eller la det være en sammenhengende tekst, og la nettleser velge når det skal bryte til neste linje selv.

Reply:
Godt poeng, jeg tror denne kom med da jeg copy+pasta fra figma 🤔🍝
Men ser at den faktisk ikke lager linjeskift i dev, det var rart: 
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/37441744/195029324-e110d1a0-b6e2-432c-b718-da540bcefe20.png">
 
Kan uansett fjerne det 👍🏼